config/locales/en.yml

```diff
@@ -218,6 +221,34 @@ en:
       itself. (These scores are only available for English Wikipedia articles.)
     articles_short: Articles
 
+  items:
+    assigned: Assigned Items
+    assignments_none: There are no assigned items yet.
+    available: Available Items
+    character_doc: The sum of characters added to items by enrolled editors between the start and end dates
+    cross_wiki_tracking: >
+      Mainspace pages edited between the start and end dates are listed below.
+      The dashboard pulls in contributions from the home wiki of this program, along
+      with any other wikis for which there is an Assignment or an Available Item.
+    does_not_exist: The Item doesn't exist. Create a Sandbox page and assign yourself to that to receive suggestions.
+    edited: Items Edited
+    edited_none: There are no edited items yet.
+    rating_doc: The rating an item has been assigned by WikiProjects on Wikipedia. These ratings may corresponds to much older versions, so they are just a rough indicator of item maturity.
+    tracked_doc: Only tracked items contribute to the the statistics for this program. When items are marked as untracked, they will be removed from the statistics during the next data update.
+    view_doc: The estimated views to each item since it was first edited, through the most recent stats update. You can use pageviews.toolforge.org to see the exact numbers.
+    rating_docs:
```

Comment:
These ratings are specific to Wikipedia, so I don't think any alternative message strings are needed.

All Wikidata items at this point (I think) will be unrated, but even better than translating the 'unrated' message will be (later) to not render any rating icon at all for Wikidata items.